pylintrc

```diff
@@ -249,7 +254,10 @@ generated-members=
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=125
+max-line-length=80
```

Comment:
I guess this is fine, although a significant change. I am very appreciative that the provided script should wrap the lines for me. @rwitten do you have a preference here?

Reply:
Thanks for the quick review!

I chose line length=80 because that is the google standard. I can ping the reference offline. On the other hand PEP8 standard is 79 https://peps.python.org/pep-0008/#maximum-line-length